processing/src/main/java/io/druid/query/DefaultQueryMetrics.java

```diff
@@ -66,15 +65,6 @@ public void queryType(QueryType query)
   }
 
   @Override
-  public void interval(QueryType query)
```

Comment:
@leventov since I'm not familiar with druid's metrics system, so I wonder how you are using it especially QueryMetrics. I think users need to consider intervals of query's data source as well to see what's really happening. Would you tell me some examples?